modules/op-release-notes-1-8.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-8_{context}"]
+= Release notes for {pipelines-title} General Availability 1.8
+
+With this update, {pipelines-title} General Availability (GA) 1.8 is available on {product-title} 4.10, and is planned to be available on {product-title} 4.11 and 4.12.
+
+[id="new-features-1-8_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
+
+:FeatureName: Running {pipelines-title} on ARM hardware
+[id="running-pipelines-on-arm-hardware-1-8_{context}"]
+=== {FeatureName}
+
+With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift Container Platform cluster that is running on ARM hardware.
+
+include::snippets/technology-preview.adoc[]
+
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+* Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
++
+This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. The `operator.tekton.dev/v1alpha1` API version is not deprecated and still used by the Operator.
++
+Bug fixes and support for the `tekton.dev/v1alpha1` API version are provided only through the end of the current GA 1.8 lifecycle. Use the `tekton.dev/v1beta1` API version instead.
+//  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
+
+[id="known-issues-1-8_{context}"]
+== Known issues
+
+* There is currently a known issue: The `s2i-nodejs` pipeline cannot use the `nodejs:14-ubi8-minimal` image stream to perform source-to-image (S2I) builds. Doing so produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message.
++
+Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image stream. You can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
```

Comment:
`oc edit` is overkill. Default value of `VERSION` parameter works fine so this known issue applies only to customers who explicitly try to use `nodejs:14-ubi8-minimal`. IMHO we can assume that they know how to "Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image stream"

Reply:
Fixed. Thank you. Please approve now.